### ISSUE TITLE
FileNotFoundError version.txt

### ISSUE THREAD

**Issue by jaraco** [python/importlib_resources#55]
In GitLab by @ipmb on May 15, 2018, 13:04

Something in 0.6 broke the ability to use `shiv` for me:

```
shiv -e saltdash:config.django_manage -o dist/saltdash-0.9.3+1c7be483d85e9c1dcafbf8014d1d7c1382007f30-py36-linux-x86_64.pyz .
Traceback (most recent call last):
  File "/root/.local/lib/python3.6/site-packages/importlib_resources/_py3.py", line 122, in open_text
    full_path, mode='r', encoding=encoding, errors=errors)
FileNotFoundError: [Errno 2] No such file or directory: '/root/.local/lib/python3.6/site-packages/importlib_resources/version.txt'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/.local/bin/shiv", line 7, in <module>
    from shiv.cli import main
  File "/root/.local/lib/python3.6/site-packages/shiv/cli.py", line 1, in <module>
    import importlib_resources  # type: ignore
  File "/root/.local/lib/python3.6/site-packages/importlib_resources/__init__.py", line 21, in <module>
    __version__ = read_text('importlib_resources', 'version.txt')
  File "/root/.local/lib/python3.6/site-packages/importlib_resources/_py3.py", line 160, in read_text
    with open_text(package, resource, encoding, errors) as fp:
  File "/root/.local/lib/python3.6/site-packages/importlib_resources/_py3.py", line 136, in open_text
    raise FileNotFoundError(message)
FileNotFoundError: 'version.txt' resource not found in 'importlib_resources'
make: *** [dist/saltdash-0.9.3+1c7be483d85e9c1dcafbf8014d1d7c1382007f30-py36-linux-x86_64.pyz] Error 1
```

**Comment by jaraco** [comment]
In GitLab by @ipmb on May 15, 2018, 13:07

Ahh, looks like `version.txt` is in the repo, but not included in the package data.

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:01

This is weird.  If I `pip install importlib_resources` then the `version.txt` file doesn't get written to the `site-packages/importlib_resources` directory.  But if I download the `tar.gz` and `pip install` from the unpacked directory, then the file exists and it works.  Hmm.

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:35

`setup.cfg` metadata issues.  I'll push out an 0.7 asap.

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:35

assigned to @warsaw

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:38

mentioned in commit bf53e07c4ce8e5cdbd501be5e5c0cc3807359ab6

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:38

mentioned in merge request !59

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:46

closed via merge request !59

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:46

closed via commit 6f9671ad1cd6674001d13d91e6171f75fa8381e1

**Comment by jaraco** [comment]
In GitLab by @warsaw on May 15, 2018, 15:46

closed via commit bf53e07c4ce8e5cdbd501be5e5c0cc3807359ab6
